common/src/main/scala/explore/model/AppContext.scala

```diff
@@ -50,20 +51,22 @@ case class AppContext[F[_]](
   def replacePage(appTab: AppTab, programId: Program.Id, focused: Focused): Callback =
     setPageVia(appTab, programId, focused, SetRouteVia.HistoryReplace)
 
-  given WebSocketClient[F, ObservationDB]     = clients.odb
-  given WebSocketClient[F, UserPreferencesDB] = clients.preferencesDB
-  given TransactionalClient[F, ITC]           = clients.itc
-  given TransactionalClient[F, SSO]           = clients.sso
+  given WebSocketJSClient[F, ObservationDB]     = clients.odb
+  given WebSocketJSClient[F, UserPreferencesDB] = clients.preferencesDB
+  given FetchJSClient[F, ITC]                   = clients.itc
+  given WebSocketJSClient[F, SSO]               = clients.sso
```

Comment:
We may want to switch this one to `FetchJSClient` too but can we reset the header already?

Reply:
Yes, we should be able to by using an additional `modParams` parameter present in all queries. Let's attempt to do that in the next PR.